views/vm/create.erb

```diff
@@ -66,6 +66,9 @@
                   }
                 ) %>
               </div>
+              <div class="sm:col-span-6">
```

Comment:
Furkan tried half width private subnet select and ipv4 checkbox. It's not looking good.
```suggestion
              <div class="col-span-full">
```